### PR TITLE
Accessibility issue fixes 8856966, 8853539

### PR DESCRIPTION
**Pull request by v-baambati**
1. BUG 8853539 : Added benchmark data to aria label
Bench Mark - aria label

2. BUG 8856966 : removed hideTooltip prop from stacked bar chart examples 
stackedbarchartissueFix

3.BUG 8845708: Added tool tip information to aria label when we have custom accessibility value
arialabel

### CODE REVIEW

File: packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx [hunk 1]
```diff
@@ -422,10 +422,14 @@ export class HorizontalBarChartBase extends React.Component<IHorizontalBarChartP
 
   private _getAriaLabel = (point: IChartDataPoint): string => {
     const legend = point.xAxisCalloutData || point.legend;
+    const benchMark = point.data;
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx:425]: benchmark is an optional property. Announce the benchmark only if present.

Comment (v-baambati) [reply packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx:425]: Added condition check in next line  " + (benchMark ? 'with benchmark at ' + `${benchMark}, ` : '')"

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

File: packages/react-examples/src/react-charting/StackedBarChart/StackedBarChart.CustomAccessibility.Example.tsx [hunk 2]
```diff
@@ -35,12 +35,7 @@ export class StackedBarChartCustomAccessibilityExample extends React.Component<{
       <>
         <StackedBarChart data={data0} href={'https://developer.microsoft.com/en-us/'} ignoreFixStyle={false} />
         <br />
-        <StackedBarChart
-          data={data1}
-          href={'https://developer.microsoft.com/en-us/'}
-          ignoreFixStyle={true}
-          hideTooltip={true}
-        />
+        <StackedBarChart data={data1} href={'https://developer.microsoft.com/en-us/'} ignoreFixStyle={true} />
```
Comment (AtishayMsft) [inline packages/react-examples/src/react-charting/StackedBarChart/StackedBarChart.CustomAccessibility.Example.tsx:38]: For this example create a toggle to turn hideTooltip option on and off.

File: packages/react-examples/src/react-charting/StackedBarChart/StackedBarChart.Basic.Example.tsx [hunk 3]
```diff
@@ -39,7 +39,6 @@ export class StackedBarChartBasicExample extends React.Component<{}, {}> {
           data={data1}
           href={'https://developer.microsoft.com/en-us/'}
           ignoreFixStyle={true}
-          hideTooltip={true}
```
Comment (AtishayMsft) [inline packages/react-examples/src/react-charting/StackedBarChart/StackedBarChart.Basic.Example.tsx:42]: For this example create a toggle to turn hideTooltip option on and off.

File: packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx [hunk 4]
```diff
@@ -422,10 +422,14 @@ export class HorizontalBarChartBase extends React.Component<IHorizontalBarChartP
...
     const yValue =
       point.yAxisCalloutData ||
       (point.horizontalBarChartdata ? `${point.horizontalBarChartdata.x}/${point.horizontalBarChartdata.y}` : 0);
-    return point.callOutAccessibilityData?.ariaLabel || (legend ? `${legend}, ` : '') + `${yValue}.`;
+    return (
+      point.callOutAccessibilityData?.ariaLabel ||
+      (legend ? `${legend}, ` : '') + `${yValue}.` + (benchMark ? 'with benchmark at ' + `${benchMark}, ` : '')
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx:431]: Also need to add benchmark announcement for StackedBarChart.

**Comment by AtishayMsft** [comment]
changefile is missing.

File: change/@fluentui-react-charting-c9360080-1be3-4b4a-a94d-755c32596050.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-charting-c9360080-1be3-4b4a-a94d-755c32596050.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31228&commitId=22f0493d589c75da73fb5a11a300db4f559818d0&env=prod&clientType=FLUENTUI)


  <details><summary><b>Pivot - Overflow</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx [hunk 6]
```diff
@@ -1048,16 +1048,16 @@ export class VerticalStackedBarChartBase extends React.Component<
           return `${legend}, ${yValue}.`;
         })
         .join(' ');
-      return (
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx:1051]: why changing the default behavior

**Comment by AtishayMsft** [comment]
ERR! /mnt/vss/_work/1/s/packages/react-examples/src/react-charting/VerticalStackedBarChart/VerticalStackedBarChart.CustomAccessibility.Example.tsx
ERR!   139:1  error  This line has a length of 124. Maximum allowed is 120  @fluentui/max-len
ERR!   150:1  error  This line has a length of 124. Maximum allowed is 120  @fluentui/max-len
ERR!   164:1  error  This line has a length of 125. Maximum allowed is 120  @fluentui/max-len

File: packages/react-examples/src/react-charting/VerticalStackedBarChart/VerticalStackedBarChart.CustomAccessibility.Example.tsx [hunk 7]
```diff
@@ -134,15 +134,23 @@ export class VerticalStackedBarChartCustomAccessibilityExample extends React.Com
             { y: 10, legend: 'Recommended Builds', color: DefaultPalette.redDark },
           ],
         }),
-        stackCallOutAccessibilityData: { ariaLabel: 'Bar stack series 1 of 6' },
+        stackCallOutAccessibilityData: {
+          ariaLabel:
+            'Bar stack series 1 of 6, 0 MetaDate1 61% MetaData2 8% MetaDate3 31% ' +
```
Comment (AtishayMsft) [inline packages/react-examples/src/react-charting/VerticalStackedBarChart/VerticalStackedBarChart.CustomAccessibility.Example.tsx:139]: Bar stack series 1 of 3 and so on below

File: packages/react-examples/src/react-charting/VerticalStackedBarChart/VerticalStackedBarChart.CustomAccessibility.Example.tsx [hunk 8]
```diff
@@ -134,15 +134,23 @@ export class VerticalStackedBarChartCustomAccessibilityExample extends React.Com
...
         chartData: secondChartPoints,
         xAxisPoint: 20,
         ...(showLine && {
           lineData: [{ y: 33, legend: 'Supported Builds', color: DefaultPalette.magentaLight }],
         }),
-        stackCallOutAccessibilityData: { ariaLabel: 'Bar stack series 2 of 6' },
+        stackCallOutAccessibilityData: {
+          ariaLabel:
+            'Bar stack series 2 of 6, 0 MetaDate1 61% MetaData2 8% MetaDate3 31% ' +
+            'Recommended Builds 10 Supported Builds 42',
```
Comment (AtishayMsft) [inline packages/react-examples/src/react-charting/VerticalStackedBarChart/VerticalStackedBarChart.CustomAccessibility.Example.tsx:152]: This aria label is incorrect.
Bar stack series 2 of 6, MetaDate1 33% MetaData2 22% MetaDate3 45% ' +
            'Supported Builds 33'

File: packages/react-examples/src/react-charting/VerticalStackedBarChart/VerticalStackedBarChart.CustomAccessibility.Example.tsx [hunk 9]
```diff
@@ -134,15 +134,23 @@ export class VerticalStackedBarChartCustomAccessibilityExample extends React.Com
...
       {
         chartData: secondChartPoints,
         xAxisPoint: 20,
         ...(showLine && {
           lineData: [{ y: 33, legend: 'Supported Builds', color: DefaultPalette.magentaLight }],
         }),
-        stackCallOutAccessibilityData: { ariaLabel: 'Bar stack series 2 of 6' },
+        stackCallOutAccessibilityData: {
+          ariaLabel:
+            'Bar stack series 2 of 6, 0 MetaDate1 61% MetaData2 8% MetaDate3 31% ' +
```
Comment (AtishayMsft) [inline packages/react-examples/src/react-charting/VerticalStackedBarChart/VerticalStackedBarChart.CustomAccessibility.Example.tsx:151]: what is the use of value 0 in the aria label

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**